cockroachcloud/frequently-asked-questions.md

```diff
@@ -17,6 +17,10 @@ This page answers the frequently asked questions about {{ site.data.products.ser
 
 {{ site.data.products.dedicated }} provides fully-managed, single-tenant CockroachDB clusters with no shared resources. {{ site.data.products.dedicated }} supports single and multi-region clusters in AWS and GCP.
 
+{{site.data.alerts.callout_success}}
```

Comment:
This probably shouldn't be in the Dedicated FAQ, but in `serverless-faqs.md`.

Reply:
Done.